### ISSUE TITLE
ctop looses connection to docker during docker restart

### ISSUE THREAD

**Issue by EHJ-52n** [bcicen/ctop#140]
Hello,
when restarting my docker service, the running ctop instance is loosing the connection to docker and shows nothing or outdated information.
The connection loss should be displayed and some reconnect procedure might be implemented.

My set-up in the following:

- **os**: `Debian GNU/Linux 9.5 (stretch)`
- **docker**:
  ```
  Client:
   Version:           18.06.0-ce
   API version:       1.38
   Go version:        go1.10.3
   Git commit:        0ffa825
   Built:             Wed Jul 18 19:09:33 2018
   OS/Arch:           linux/amd64
   Experimental:      false
  
  Server:
   Engine:
    Version:          18.06.0-ce
    API version:      1.38 (minimum version 1.12)
    Go version:       go1.10.3
    Git commit:       0ffa825
    Built:            Wed Jul 18 19:07:38 2018
    OS/Arch:          linux/amd64
    Experimental:     false
  ```
- **ctop**: `ctop version 0.7.1, build 5261444 go1.10` 

Do you need any additional information?

**Comment by roja6969** [comment]
Is your comment why is it that a dependant service stops working when the independent service is no longer available. If you launched it from a container "all" that is required to run it is shutdown once the docker instance is off/restarted the process is dead there is nothing to restart it. /var/run/docker.sock is gone playtime is over.

**Comment by bcicen** [comment]
@EHJ-52n while many times a broken connection may be unrecoverable, this can be bothersome during restarts or under poor network conditions. I've added retry logic and disconnect screen in https://github.com/bcicen/ctop/commit/98fcfe8b6f3317e04339745a77a02bc13bba1240

**Comment by EHJ-52n** [comment]
@bcicen I am sorry, but I do not have time for building ctop myself.

Do you have any release candidate, beta, alpha, whatever, which I can try?

I just updated my local ctop to version 0.7.2, build 70bd2ae go1.11.3 and the issue is still there (as expected).

**Comment by bcicen** [comment]
@EHJ-52n you can use the ["latest" Docker image](https://github.com/bcicen/ctop#docker), which is always up to date with the latest master.

If this is not an option, it will be included in the 0.7.3 release shortly.
